include/sway/config.h

```diff
@@ -163,6 +166,14 @@ struct bar_config {
 		char binding_mode_border[10];
 		char binding_mode_bg[10];
 		char binding_mode_text[10];
+
+		bool has_focused_background;
+		bool has_focused_statusline;
+		bool has_focused_separator;
+
+		bool has_binding_mode_border;
+		bool has_binding_mode_bg;
+		bool has_binding_mode_text;
```

Comment:
Is there any way we could encode these into the char arrays? i.e. "inherit" or something.